src/GeositeFramework/css/app.css

```diff
@@ -262,8 +262,8 @@ input.code-like, textarea.code-like {
             }
         .content .map .tools {
             position: absolute;
-            top: 92px;
-            left: 20px;
+            top: 75px;
+            right: 20px;
             z-index: 1;
```

Comment:
I didn't move those because they are part of the ESRI map not plugins, but I guess it does make sense to move them too.